README.md

```diff
@@ -1,10 +1,10 @@
-<p align="center">
+<div align="center">
   <a href="https://sentry.io/?utm_source=github&utm_medium=logo" target="_blank">
-    <img src="https://sentry-brand.storage.googleapis.com/sentry-wordmark-dark-280x84.png" alt="Sentry" width="280" height="84">
+      <img src="https://sentry-brand.storage.googleapis.com/github-banners/github-sdk-laravel.png" alt="Sentry for Apple">
```

Comment:
I don't think you want Laravel here 😆
It's https://sentry-brand.storage.googleapis.com/github-banners/github-sdk-cocoa.jpg